examples/taptree_of_horror/taptree_of_horror.rs

```diff
@@ -0,0 +1,314 @@
+    let eve = format!("tprv8ZgxMBicQKsPdYD5umCPZeh6tMqKfQATctqJbycgJ5N1rrJ15cHMgxds5iYENHZHmkMiXccAqUFx2k3ZNwU9qxPMjrKvTbCtgLFxk7mjMWD/{normal_path}/*");
+    let frank = format!("tprv8ZgxMBicQKsPemyPyxqZ85T1UjpToCbLQ7uSn4JpbtwMBCodjvrLbgjBZeGgT4tMHdHCqyieDwCNzE7RrtRMVCQjPKQbGJzrg5vfn4eT7og/{normal_path}/*");
+    let heather = format!("tprv8ZgxMBicQKsPefp99xLkwnQbU9LEEb8v4Aig3o4hwnjbaYotixkbJv3Ssmog68ptHij2LgExefNU96DYJKtFbDazTr1jm48twYhQLG775qw/{normal_path}/*");
+    let ian = format!("tprv8ZgxMBicQKsPfGsuwfAdg3xPP452wreLk7ZEgusb8zdMqh1fyKGKnzFbxyxeHY3qhg8ESDRp5F6RgWiQGcvkmLyERcMys5V8DuT4gvxMDmS/{unhardened_path}/*");
+    let judy = format!("tprv8ZgxMBicQKsPcz4VcN87e2e9k1LHDBLLajbVSAKAedpm1qakjtRT5xrdnmsQARWAfwg3REr6sNd5YHeWuWkHVvyey3rYmq9xYorMvwY3XAB/{normal_path}/*");
+    let liam = format!("tprv8ZgxMBicQKsPcycCJ2v7B1utZrhWJNdQTBm7m9eR6iX1D9a9YvjbCNeT6dTEdMh4JziVCHD4YHQ7AXkZNMLfaBVf3CCiVWQLxwdU2SnrPcT/{normal_path}/*");
+    let s_backup_1 = format!("tprv8ZgxMBicQKsPetPYYt5GUtNmQPChghNDBLbgJXz3cTZopeDrxHMendLpujaBHMPX3dXLZcv2NkgAvxMeuf1jWBU3iYxdeJAhktdeM9cKcYF/{normal_path}/*");
+    let x_backup_2 = format!("tprv8ZgxMBicQKsPdf55kYg8pVPrUaW3VLZyt8XwxwrvSPkpskxDah1mAWypsTTZJeomWPrGf4e5RyT4zVzENHrwAsXxJP2EPYbfrYLRVFC1rLb/{normal_path}/*");
+
+    // define DescriptorSecretKeys
+    let internal_desc_secret = DescriptorSecretKey::from_str(&internal).unwrap();
+    let a_descriptor_desc_secret = DescriptorSecretKey::from_str(&alice).unwrap();
+    let b_descriptor_desc_secret = DescriptorSecretKey::from_str(&bob).unwrap();
+    let c_descriptor_desc_secret = DescriptorSecretKey::from_str(&charlie).unwrap();
+    let d_descriptor_desc_secret = DescriptorSecretKey::from_str(&dave).unwrap();
+    let e_descriptor_desc_secret = DescriptorSecretKey::from_str(&eve).unwrap();
+    let f_descriptor_desc_secret = DescriptorSecretKey::from_str(&frank).unwrap();
+    let h_descriptor_desc_secret = DescriptorSecretKey::from_str(&heather).unwrap();
+    let i_descriptor_desc_secret = DescriptorSecretKey::from_str(&ian).unwrap();
+    let j_descriptor_desc_secret = DescriptorSecretKey::from_str(&judy).unwrap();
+    let l_descriptor_desc_secret = DescriptorSecretKey::from_str(&liam).unwrap();
+    let s_descriptor_desc_secret = DescriptorSecretKey::from_str(&s_backup_1).unwrap();
+    let x_descriptor_desc_secret = DescriptorSecretKey::from_str(&x_backup_2).unwrap();
+    let grim = produce_grim_hash("sovereignty through knowledge");
+    let kelly = produce_kelly_hash("the ultimate pre-preimage");
+
+    // define keys that will be used for the policy and for signing.
+    let internal_xpub: miniscript::DescriptorPublicKey =
+        internal_desc_secret.to_public(secp).unwrap();
+    let _intneral_xpriv: DescriptorXKey<Xpriv> = match internal_desc_secret.clone() {
+        DescriptorSecretKey::XPrv(x) => Some(x),
```

Comment:
In 1282d30653a3cb4e188e59d5a3dd8de909aa9477:

I would change the spelling of `intneral_xpriv`, move the `.clone` into the `Some`, and comment this whole statement out (since the place that it's used is also commented out).

In general I think it's fine to have commented-out-code in examples. cc @sanket1729 what do you think?